src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/TextBoxTests/TextBoxTests.cs

```diff
@@ -194,9 +194,16 @@ public async Task TextBox_Readonly()
 			await Task.Delay(100);
 			_app.WaitForText(txt, initialText + "Hello!");
 
-			tglReadonly.FastTap();
+			var previousText = txt.GetDependencyPropertyValue<string>("Text");
+
+			tglReadonly.Tap();
 			_app.EnterText(txt, " Works again!");
-			_app.WaitForText(txt, initialText + "Hello! Works again!");
+
+			var newText = "";
+
+			_app.WaitFor(() => (newText = txt.GetDependencyPropertyValue<string>("Text")) != previousText);
+
+			Assert.IsTrue(newText.Contains("Works again!"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this call to 'newText.Contains' to an overload that accepts a 'StringComparison' as a parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5118960)